tensionmap.py

```diff
@@ -179,10 +179,8 @@ def tm_update(obj, context):
             group_stretch.add(add_index, stretch_value, "REPLACE")
 
         if obj.data.tm_enable_vertex_colors:
-            # red
-            vertex_colors[i * number_of_tm_channels] = stretch_value
-            # green
-            vertex_colors[i * number_of_tm_channels + 1] = squeeze_value
+            # red, green, blue, alpha
+            vertex_colors[i] = (stretch_value,squeeze_value, 0.0, 1.0)
```

Comment:
Do use the formatter I was talking about in my other comment. Because here also, it is missing a space between the first two elements of the tuple.

I don't like nit picking about code styling yet I like it to be consistent, that's why I prefer having one formatter dictate the format of the whole code and never worry about it again.

I'll add in the README the formatter I use so that contributors know what style is expected.

Reply:
Yeah, I'll add a formatter.
Style has to be consistent and having to check and correct it manually each time is essentially wasting both of our time. :)